certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7809/CertificateModelFactoryFK7809.java

```diff
@@ -162,9 +177,61 @@ public class CertificateModelFactoryFK7809 implements CertificateModelFactory {
   private static final FieldId DIAGNOS_5 = new FieldId("diagnos5");
   private static final short DIAGNOSIS_CODE_LIMIT = (short) 81;
 
+  private static final ElementId FUNKTIONSNEDSATTNING_CATEGORY_ID = new ElementId("KAT_5");
+  private static final ElementId FUNKTIONSNEDSATTNING_ID = new ElementId("funktionsnedsattning");
+  private static final FieldId FUNKTIONSNEDSATNING_FIELD_ID = new FieldId(
+      "funktionsnedsattning");
+  private static final FieldId FUNKTIONSNEDSATTNING_INTELLEKTUELL_FUNKTION_ID = new FieldId("8.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_KOMMUNIKATION_SOCIAL_INTERAKTION_ID = new FieldId(
+      "9.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_UPPMAKRMSAHET_ID = new FieldId("10.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_PSYKISK_FUNKTION_ID = new FieldId("11.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_HORSELFUNKTION_ID = new FieldId("48.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_SYNFUNKTION_ID = new FieldId("49.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_SINNESFUNKTION_ID = new FieldId("12.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_KOORDINATION_ID = new FieldId("13.2");
+  private static final FieldId FUNKTIONSNEDSATTNING_ANNAN_KROPPSILIG_FUNKTION_ID = new FieldId(
```

Comment:
I was first confused why I could not find a 8.2 value in the XML, then I realised its a fieldId and not elementId.

To me it would make more sense to call these "json-id:s" something like "annanKroppsiligFunktion" instead of a traditional questionId. But this is very subjective, and I dont know if we have established a pattern for this yet. If you agree with me, update it... if not, dont mind this comment! :)

Reply:
I'll leave them as they are and we can refactor if we decide to set it as a rule for CS